src/components/CollectionDetailPage/CollectionItem/CollectionItem.tsx

```diff
@@ -57,7 +57,7 @@ export default class CollectionItem extends React.PureComponent<Props> {
         {isFree(item) ? (
           t('global.free')
         ) : (
-          <Mana className={styles.mana} network={Network.MATIC}>
+          <Mana showTooltip={true} className={styles.mana} network={Network.MATIC}>
```

Comment:
This can also be written as:
```suggestion
          <Mana className={styles.mana} network={Network.MATIC} showTooltip>
```

Reply:
Would you mind changing it also in the other places (also in the [other repo](https://github.com/decentraland/marketplace/pull/1532))?